packages/app-store/around/components/icon.tsx

```diff
@@ -0,0 +1,33 @@
+export default function Icon() {
```

Comment:
Why a tsx? make it a normal SVG, put it in static folder and server it with `/api/app-store/around/icon.svg`

Reply:
@hariombalhara we will need a tour to update the CLI. There are multiple improvements we can make.

Reply:
Sure. Should we do a call that can be recorded? or I can record a loom.